### DISCUSSION TITLE
same issue with #3750 which was closed (but the server version is 2.25.0)

### DISCUSSION THREAD

**Discussion by qhgary** [dani-garcia/vaultwarden#4302] [Q&A]
My server side version is 2.25.0 because I pulled with the "latest" label. However, the same issue(same wrong message) occurred when I tried to use bitwarden client side, including browser plugin, android app, etc.
By the way, everything goes well when I pulled the version 1.29.1

I use self-signed certificate and local DNS service， and it goes well with other service， so it should have no problem.

**Comment by BlackDex** [comment]
The current latest version is 1.30.2, so you are behind with your version.

If you tried to pull the latest tagged image but you receive a version which has web-vault 2.25.0, then either your docker mirror doesn't have an updated version. Or maybe your docker version is too old. You need at least v20.10 of docker if I'm correct.

**Reply by stefan0xC** [reply]
Or you might even need to switch from `bitwardenrs/server:latest` to `vaultwarden/server:latest`.

**Reply by BlackDex** [reply]
Indeed!
